src/app/layouts/base-layout/components/partnered/partnered.tsx

```diff
@@ -8,10 +8,18 @@ export const Partnered = () => (
     <div className="partnered content">
       <WidgetTitle size="small">Partnered with</WidgetTitle>
       <div className="partners">
-        <EpamLogo />
-        <JetBrainsLogo />
-        <AwsLogo />
-        <GithubLogo />
+        <figure className="partner-logo-container">
+          <EpamLogo />
+        </figure>
+        <figure className="partner-logo-container">
+          <JetBrainsLogo />
+        </figure>
+        <figure className="partner-logo-container">
+          <AwsLogo />
+        </figure>
+        <figure className="partner-logo-container">
+          <GithubLogo />
+        </figure>
```

Comment:
I wouldn't use the `figure` tag here

Reply:
If I remember correctly, there was a plan to unify icons in favor of svg and remove this icon components. Then figure with figcapture will work nice.

Reply:
@SpaNb4 So what should we do? I can change it. 
The reason I chose "figure", is because I just copied the existing code
https://github.com/rolling-scopes/site/blob/907cb65cc7bd5566b2d70ee8a13c6e89487791bd/src/widgets/alumni/ui/alumni.tsx#L37

Reply:
> @SpaNb4 So what should we do? I can change it. The reason I chose "figure", is because I just copied the existing code
> 
> https://github.com/rolling-scopes/site/blob/907cb65cc7bd5566b2d70ee8a13c6e89487791bd/src/widgets/alumni/ui/alumni.tsx#L37

Please remove it

Reply:
In this case, I'm not able to remove the warper, but I changed it to div.
https://github.com/rolling-scopes/site/pull/433/commits/b4f95962a4496573ef43bd97699d77771c0d8836